Rakefile

```diff
@@ -2,6 +2,7 @@ require 'html-proofer'
 
 task :test do
   sh "bundle exec jekyll build"
-  options = { :assume_extension => true }
+  options = { :check_html => true, # :validation => { :report_missing_names, :report_invalid_tags => true, :report_script_embeds => true },
```

Comment:
yes because there are a ton more errors but I wanted to document those extra options